styles/_status-label.scss

```diff
@@ -51,7 +51,8 @@
     color: $secondary-text-color;
   }
 
-  &-pending {
+  &-pending,
```

Comment:
Giving that we are changing from `status-label-pending` to `status-label-pending-moderation`, do we still need to target `status-label-pending`?

Reply:
Yes, because the whole point of the rename is that status-label-pending and status-label-pending-moderation are different things (that happen to have identical rendering).

The former relates to lotus requests; the latter relates to guide community posts.

Reply:
Gotcha! Couldn't find a reference for `.status-label-pending`, but we have it indeed for requests: https://github.com/zendesk/copenhagen_theme/blob/master/templates/requests_page.hbs#L125

Reply:
`status-label-pending` is referenced purely from the templates within this theme.